app/controllers/application_controller.rb

```diff
@@ -153,6 +155,10 @@ def pundit_user
 
   private
 
+  def set_time_zone(&block) # rubocop:disable Naming/AccessorMethodName
```

Comment:
@harigopal: Shouldn't we enable this cop back?

Reply:
End-of-line comments like these disable cops on a single line.

See https://docs.rubocop.org/en/stable/configuration/#disabling-cops-within-source-code